src/components/DealTranchesStep.tsx

```diff
@@ -0,0 +1,77 @@
+import { FunctionComponent } from "react";
+import { useIntl } from "react-intl";
+import { classNames } from "@utils/format.utils";
+import { Form, FormInstance, Radio } from "antd";
+import { FormItem } from "./FormItem";
+import { SelectorCard } from "./SelectorCard";
+import { Button } from "./Button";
+import { Icon, IconDimension } from "./Icon";
+import { defaultTranches } from "@consts";
+import { TrancheOption } from "./TrancheOption";
+
+interface DealTranchesStepProps {
+	className?: string;
+	form: FormInstance;
+	setCurrentStep: (step: number) => void;
+}
+
+export const DealTranchesStep: FunctionComponent<DealTranchesStepProps> = ({
+	className,
+	form,
+	setCurrentStep,
+}) => {
+	const selectedTranche = Form.useWatch("trancheStructure", form);
+	const intl = useIntl();
+
+	className = classNames([className, "space-y-8"]);
```

Comment:
I know it's not part of this PR persé but could you maybe add a comment to this helper function explaining why we introduced it?